spec/namespaces/ism.yaml

```diff
@@ -84,36 +162,160 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/ism.change_policy@200'
+  /_plugins/_ism/explain:
+    get:
+      operationId: ism.explain.0
+      x-operation-group: ism.explain
+      description: Gets the currently applied policy on all indices.
+      externalDocs:
+        url: https://opensearch.org/docs/latest/im-plugin/ism/api/#explain-index
+      requestBody:
+        $ref: '#/components/requestBodies/ism.explain'
+      responses:
+        '200':
+          $ref: '#/components/responses/ism.explain@200'
+    post:
+      operationId: ism.explain.1
+      x-operation-group: ism.explain
+      description: Gets the currently applied policy on all indices.
+      externalDocs:
+        url: https://opensearch.org/docs/latest/im-plugin/ism/api/#explain-index
+      requestBody:
+        $ref: '#/components/requestBodies/ism.explain'
+      responses:
+        '200':
+          $ref: '#/components/responses/ism.explain@200'
   /_plugins/_ism/explain/{index}:
     get:
-      operationId: ism.explain_index.0
-      x-operation-group: ism.explain_index
+      operationId: ism.explain_policy.0
+      x-operation-group: ism.explain_policy
```

Comment:
Renaming groups will result in breaking changes in the generated clients. Are you sure you wanna do this?

Reply:
Yes, it's all new code. I added ISM support that has yet to be merged into any client AFAIK.